python/triton/compiler/compiler.py

```diff
@@ -328,6 +328,8 @@ def _init_handles(self):
         if self.module is not None:
             return
         device = driver.active.get_current_device()
+        # create launcher
+        self.run = driver.active.launcher_cls(src, self.metadata)
```

Comment:
This should probably be `self.src`, right?  And maybe we should explicitly do `self.src = None` after this so we don't hang on to the source for the lifetime of CompiledKernel (which is essentially forever)... though I don't love the need for that sort of explicit resource management

Reply:
Thanks for catching it!